### PR TITLE
fix(Core/Maps): Add AREA_SILVERMOON_CITY to AreaDefines.h

### PR DESCRIPTION
**Pull request by NoxMax**
Somehow Silvermoon was overlooked in definitions. Other capitals (including Exodar) are there. Thanks @Celandriel for noticing this.

## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [ ] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes 

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [ ] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [ ] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [ ] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by heyitsbench** [comment]
It wasn't added because it isn't used.

**Comment by NoxMax** [comment]
Should it be added on the account of it being a capital for future use, or is it not to be added until explicitly needed?

**Comment by heyitsbench** [comment]
> Should it be added on the account of it being a capital for future use, or is it not to be added until explicitly needed?

IMO, not until needed, else we'd fill the entire AreaTable DBC into this header.

**Comment by NoxMax** [comment]
Alright. I will keep this PR open for a couple of days, but if that's the general consensus then I will close it by then.

**Comment by FrancescoBorzi** [comment]
IMHO it doesn't harm to add it even if not used yet, but I don't have a strong opinion on the matter.

File: src/server/game/Maps/AreaDefines.h [hunk 1]
```diff
@@ -96,6 +96,7 @@ enum AreaTableIDs : uint32
     AREA_EVERSONG_WOODS                                      = 3430,
     AREA_GHOSTLANDS                                          = 3433,
     AREA_HELLFIRE_PENINSULA                                  = 3483,
+    AREA_SILVERMOON_CITY                                     = 3487;
```
Comment (sogladev) [inline src/server/game/Maps/AreaDefines.h]:
```suggestion
    AREA_SILVERMOON_CITY                                     = 3487,
```

I don't mind adding entries to this if someone feels they need them

**Review by sogladev — CHANGES_REQUESTED** [review]

**Comment by NoxMax** [comment]
The entry for Silvermoon came about because we needed to call on the capitals over in Playerbots as opposed to using magic numbers, so I hope it is added to core even if it's not directly utilized in AC itself yet.

**Review by sogladev — APPROVED** [review]

**Comment by Nyeriah** [comment]
> The entry for Silvermoon came about because we needed to call on the capitals over in Playerbots as opposed to using magic numbers, so I hope it is added to core even if it's not directly utilized in AC itself yet.

Fine by me
